task/test-tests-scan

```diff
@@ -59,9 +59,6 @@ GITHUB_DATA = {
         "sha": "abcdef",
     },
     "/users/user/repos": [{"full_name": "project/repo"}],
-
-    # This is accessed via fetch_file()
-    "/project/repo/abcdef/.cockpit-ci/container": "supertasks",
```

Comment:
I actually realized that this would get quoted and didn't care, since I was lazy :)

Thanks for the cleanup.